### ISSUE TITLE
GET parameters not getting to elFinderConnector

### ISSUE THREAD

**Issue by ThePaavero** [barryvdh/laravel-elfinder#19]
I kept getting "Unknown command." errors in the elfinder modal. Turned out Laravel had stripped out all native $_GET keys and they were not present at the point the connector needed them (e.g. "action=open", etc.). I got it working by adding this to the controller (at row 78):

```
$url = \Request::server('REQUEST_URI');
if(strstr($url, '?'))
{
    $bits = explode('?', $url);
    $url = $bits[1];
    parse_str($url, $gets);
    $_GET = $gets;
}
```

**Comment by muratsplat** [comment]
I have same issue and @ThePaavero's solution fixes my problemm... I think that bug gets on subdomains or different ports which is not default port 80..

**Comment by barryvdh** [comment]
Are you using any other plugins or validation stuff or something?

How about something like this:

    $_GET['cmd'] = \Input::get('cmd');

Still not pretty but seems a bit less hacky..

**Comment by muratsplat** [comment]
No.. I think  this problem is trigger by wrong nginx configuration.. But I'am not sure yet..

Because our $_GET methods is not normal...

```php
[
  '?cmd' => 'open'  // Should it like this 'cmd' !!!  ?
  'target' => 'l1_dXBsb2Fkcw' 
  'init' => '1'
  'tree' =>'1'
  '_' => '1420738843491' 
]
```
If '?cmd' key changed by re-edit 'cmd', the issue fixes...

**Comment by barryvdh** [comment]
That is strange indeed. Can you try to narrow it down?
What is the exact url? Can you add the query string to some plain php file?.

eg test.php?cmd=open&target=l1_dXBsb2Fkcw&...
die(var_dump($_GET));

And if that gives you correct result, perhaps disable some plugins or put that line increasingly further in the laravel code, so you now if it changes somewhere..

**Comment by barryvdh** [comment]
Maybe a incorrect nginx rewrite configuration?

**Comment by muratsplat** [comment]
Yes you are right . My nginx configuretion is incorrect. 
problem old line in nginx conf
```sh
    location / {
        try_files $uri $uri /index.php?$is_args$args;
    }
```
I check the configuration and  was changed  it  like this
```sh
 location / {
        try_files $uri $uri/ /index.php?$query_string;
    }
```

Now elfinder works perfectly :+1: 

I belive that @ThePaavero's  nginx configuration was incorrect :d

**Comment by muratsplat** [comment]
Thanks for your care @barryvdh  :+1:

**Comment by antonaut** [comment]
Hmm.. I'll look over my configs aswell. Thanks guys! :+1:
